python/testmonitor/results/results.py

```diff
@@ -45,24 +47,15 @@ def measure_power(current: float, voltage: float = 0) -> object:
     power = current * current_loss * voltage * voltage_loss
 
     # Record electrical current and voltage as inputs.
-    inputs = [
-        NamedValueObject(name="current", value=current),
-        NamedValueObject(name="voltage", value=voltage)
-    ]
+    inputs = [NamedValueObject(name="current", value=current), NamedValueObject(name="voltage", value=voltage)]
 
     # Record electrical power as an output.
-    outputs = [
-        NamedValueObject(name="power", value=power)
-    ]
+    outputs = [NamedValueObject(name="power", value=power)]
 
     return power, inputs, outputs
 
 
-def build_power_measurement_params(
-        power: float,
-        low_limit: float,
-        high_limit: float,
-        status: object) -> object:
+def build_power_measurement_params(power: float, low_limit: float, high_limit: float, status: object) -> object:
```

Comment:
A couple more uses of `object` type annotations here - could we either be more specific, or use `Any` from `typing`?